hygiene_string.js

```diff
@@ -0,0 +1,12 @@
+conn = new Mongo();
+db = conn.getDB("weisheng");
+
+db.hygiene.find({'PHONE':{$type:18}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: ""+e.PHONE}})}
+	);
+db.hygiene.find({'PHONE':{$type:16}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: e.PHONE.toString()}})}
+	);
+db.hygiene.find({'PHONE':{$exists:true}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: "+1"+e.PHONE}})}
+	);
```

Comment:
This seems to be just formatting the phone numbers to be consistent with each other right?
I am guessing that this would leave us with the work of getting the join up. Which sounds like a great idea as now we can perform the join in mongo as opposed to storm code, which would have been slower. I understand the context of the conversation on hangout better now. Please correct me if I am wrong. Also apart from the first question, LGTM.
Nitpicky suggestion -> Add comments in the individual commands in the script to help understand the need of each one of the commands.